webpack.config.js

```diff
@@ -0,0 +1,15 @@
+var webpack = require ('webpack')
```

Comment:
`;`?

@iamakulov Maybe we should to check with eslint gulpfile and now webpack config too?